src/cdk/stepper/stepper-button.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive} from '@angular/core';
+import {CdkStepper} from './stepper';
+
+@Directive({
+  selector: 'button[cdkStepperNext]',
+  host: {
+    '(click)': '_onClick()',
```

Comment:
Instead of creating an `_onClick` handler, you can just do
```ts
host: {
  '(click)': '_stepper.next()',
}
```
(would have to make `_stepper` not private; just keep the underscore as it implies "internal"). Would condense the entire class down to
```ts
@Directive({
  selector: 'button[cdkStepperNext]',
  host: {'(click)': '_stepper.next()'}
})
export class CdkStepperNext {
  constructor(public _stepper: CdkStepper) { }
}
```